From ac9de424bc0a896e8e30b5ec176daf924a2cefb2 Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Mon, 31 Aug 2009 16:41:12 +0200
Subject: [PATCH] mixer: Add Speaker and Beep names to the weight list

Added strings "Speaker" and "Beep" to the weight list so that the entries
appear in more appropriate positions.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 src/mixer/simple_none.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/mixer/simple_none.c b/src/mixer/simple_none.c
index 4802200..9259a08 100644
--- a/src/mixer/simple_none.c
+++ b/src/mixer/simple_none.c
@@ -134,6 +134,7 @@ static int get_compare_weight(const char *name, unsigned int idx)
 	static const char *const names[] = {
 		"Master",
 		"Headphone",
+		"Speaker",
 		"Tone",
 		"Bass",
 		"Treble",
@@ -158,6 +159,7 @@ static int get_compare_weight(const char *name, unsigned int idx)
 		"I2S",
 		"IEC958",
 		"PC Speaker",
+		"Beep",
 		"Aux",
 		"Mono",
 		"Playback",
-- 
1.5.5.1

